src/components/StylesheetsContext.ts

```diff
@@ -70,12 +70,14 @@ export function useStylesheetsContextWithDocumentDefault({
   return useStylesheetsContextWithDefaultValues({
     newStylesheets,
     defaultStylesheets: [
-      // DocumentView.css is an artifact produced by webpack, see the DocumentView entrypoint included in
-      // `webpack.config.mjs`. We build styles needed to render documents separately from the rest of the sidebar
-      // in order to isolate the rendered document from the custom Bootstrap theme included in the Sidebar app
-      "/DocumentView.css",
       bootstrap,
       bootstrapOverrides,
+      // DocumentView.css and TextAreaWidget.css are artifacts produced by webpack, see the DocumentView and TextAreaWidget
+      // entrypoints included in `webpack.config.mjs`. We build styles needed to render documents separately from the rest
+      // of the sidebar in order to isolate the rendered document from the custom Bootstrap theme included in the Sidebar app
+      // place last to override the default bootstrap styles via the cascade
+      "/DocumentView.css",
+      "/TextAreaWidget.css",
```

Comment:
As suspected, this is because the component appears in a shadow DOM, so this applies:

- https://github.com/pixiebrix/pixiebrix-extension/issues/7670

Rather than building one stylesheet per widget, we should build one stylesheet per shadow DOM component.

`DocumentView` is a shadow DOM component so it's fine, and this one should be:

```suggestion
      "/EphemeralForm.css",
```

Reply:
It's not just EphemeralForm. This also applies to custom form components. 

I don't disagree that one per shadow DOM component would be better, but that will mean adding more than one css file to only pick up the styling for TextAreaWidget.

Reply:
> This also applies to custom form components.

Yeah every shadow-dom component should have its own stylesheet, instead of loading 2/3 stylesheets each.

> that will mean adding more than one css file

In this PR yes, but as widgets/components with custom style increase, so will the list of stylesheets. Almost the same, but then the next developer has to ask on Slack again why their styles aren't being loaded.

We actually already have multiple components creating and loading their own stylesheet in different ways:

https://github.com/pixiebrix/pixiebrix-extension/blob/8b1b872a66ee1194f86bfc0d8db3661533fbddb1/src/components/floatingActions/FloatingActions.tsx#L22

https://github.com/pixiebrix/pixiebrix-extension/blob/8b1b872a66ee1194f86bfc0d8db3661533fbddb1/src/components/StylesheetsContext.ts#L76

https://github.com/pixiebrix/pixiebrix-extension/blob/8b1b872a66ee1194f86bfc0d8db3661533fbddb1/src/components/quickBar/QuickBarApp.tsx#L30

https://github.com/pixiebrix/pixiebrix-extension/blob/8b1b872a66ee1194f86bfc0d8db3661533fbddb1/src/components/selectionToolPopover/SelectionToolPopover.tsx#L26-L27

https://github.com/pixiebrix/pixiebrix-extension/blob/8b1b872a66ee1194f86bfc0d8db3661533fbddb1/src/contentScript/commandPopover/CommandPopover.tsx#L30

https://github.com/pixiebrix/pixiebrix-extension/blob/8b1b872a66ee1194f86bfc0d8db3661533fbddb1/src/contentScript/selectionTooltip/SelectionToolbar.tsx#L19-L20

Reply:
Good callout. I'll fix _this_ case and create a separate devex ticket to clean all of this up.

Reply:
https://github.com/pixiebrix/pixiebrix-extension/issues/7938